Packs/CheckpointFirewall/Playbooks/Checkpoint_-_Block_IP_-_Append_Group.yml

```diff
@@ -0,0 +1,1160 @@
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "34":
+    id: "34"
+    taskid: a23a1727-94a2-4a01-82ea-55e7c5f947f6
+    type: regular
+    task:
+      id: a23a1727-94a2-4a01-82ea-55e7c5f947f6
+      version: -1
+      name: Get object group from FW
+      description: Gets full data for the specified group.
+      script: '|||checkpoint-group-get'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      identifier:
+        simple: ${inputs.group_name}
+      session_id:
+        simple: ${CheckPoint.Login.session-id}
+    continueonerror: true
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 220,
+          "y": 1870
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "35":
+    id: "35"
+    taskid: 6bdf7795-cd12-4ece-825f-701cc62b6457
+    type: regular
+    task:
+      id: 6bdf7795-cd12-4ece-825f-701cc62b6457
+      version: -1
+      name: Get current group members
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-admin/playbooks/automations.html
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "30"
+    scriptarguments:
+      key:
+        simple: currentMembers
+      value:
+        complex:
+          root: CheckPoint.Group
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: CheckPoint.Group.name
+                iscontext: true
+              right:
+                value:
+                  simple: inputs.group_name
+                iscontext: true
+              ignorecase: true
+          accessor: members.member-name
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 220,
+          "y": 1040
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 3975,
+        "width": 1550,
+        "x": -330,
+        "y": -900
+      }
+    }
+  }
+inputs:
+- key: IP
+  value: {}
+  required: true
+  description: Array of malicious IPs to block.
```

Comment:
Just array or also command delimited? provide examples.

Reply:
The context array - I'll try to add an example and check if the ["1.1.1.1","2.2.2.2"] option as an input will work (instead of providing context path like ${IP})

Reply:
Support added also for comma separated values